proxy_manager.go

```diff
@@ -55,12 +55,11 @@ func newProxyManager(bundle creationBundle) *proxyManager {
 }
 
 func (m *proxyManager) Proxies() map[string]interface{} {
-	cp := make(map[string]interface{}, len(m.proxies))
-	m.mu.RLock()
-	defer m.mu.RUnlock()
-	for k, p := range m.proxies {
-		cp[k] = p
-	}
+	cp := map[string]interface{}{}
+	m.proxies.Range(func(key, value interface{}) bool {
```

Comment:
There is no way to get the length of `sync.Map`, right?

Reply:
There's no `Len` method, but you can range over it to get the length, which is not useful in this case.